src/Carrier.php

```diff
@@ -51,6 +56,7 @@ class Carrier
         self::CODE_FEDEX => 'FedEx',
         self::CODE_UPS => 'UPS',
         self::CODE_USPS => 'USPS',
+        self::CODE_LASER_SHIP => 'Laser Ship',
```

Comment:
@bell2041 It looks like they go by `LaserShip` (no space). Mind making that small change before we merge this in?